cli/cache/emit.rs

```diff
@@ -16,16 +16,38 @@ pub struct EmitCache {
   disk_cache: DiskCache,
   emit_failed_flag: AtomicFlag,
   file_serializer: EmitFileSerializer,
+  behavior: Behavior,
+}
+
+#[derive(Debug)]
+enum Behavior {
+  Normal,
+  Disable,
 }
 
 impl EmitCache {
   pub fn new(disk_cache: DiskCache) -> Self {
+    let behavior = match std::env::var("DENO_EMIT_CACHE_BEHAVIOR")
```

Comment:
`MODE` instead of `BEHAVIOR` because it doesn't have regional spellings and is shorter?